extensions/amp-subscriptions-google/0.1/amp-subscriptions-google.js

```diff
@@ -183,6 +198,7 @@ export class GoogleSubscriptionsPlatform {
     this.serviceAdapter_.resetPlatforms();
   }
 
+  /* TODO(jpettitt): contribute? */
```

Comment:
Is the behavior here confirmed though? I.e. is that a `TODO` for the next PR or unconfirmed feature yet?